.circleci/config.yml

```diff
@@ -783,7 +763,9 @@ workflows:
           - not:
               equal: ["UNKNOWN", << pipeline.parameters.study_key >>]
       jobs:
-        - app-run-tests-job
+        - app-ui-unit-test:
+            study_key: << pipeline.parameters.study_key >>
+        - toolkit-sdk-unit-test
```

Comment:
Tested in [CircleCI](https://app.circleci.com/pipelines/github/broadinstitute/ddp-angular/23055/workflows/daae66ea-0182-42e1-a792-66dbc122a4be).
```
./run_ci.sh run-tests singular pepper-298-ci-config-refactor-2 dev
Will try to initiate build from branch: pepper-298-ci-config-refactor-2
{
  "number" : 23055,
  "state" : "pending",
  "id" : "c23a648d-e9c1-4d32-a18c-ea8e2c3c2d36",
  "created_at" : "2023-02-10T00:12:08.520Z"
}
```

<img width="827" alt="Screenshot 2023-02-09 at 7 16 25 PM" src="https://user-images.githubusercontent.com/35533885/217968384-1ac15925-16e8-40e6-aad3-47036def5f1d.png">